common/http/platform/beast/http.cpp

```diff
@@ -794,6 +794,22 @@ void Stream::CallErrorHandler(
 	server_.RemoveStream(shared_from_this());
 }
 
+void Stream::CallErrorHandler(
+	const error_code &ec, const IncomingRequestPtr &req, IdentifiedRequestHandler handler) {
+	CallErrorHandler(error::Error(ec.default_error_condition(), ""), req, handler);
```

Comment:
Q: Why don't we extract the `message`?

Reply:
We do, it's in the `ec.default_error_condition()`, which, when later using `String()`, calls `message()`.